tests/test_dataload.py

```diff
@@ -0,0 +1,34 @@
+from django.test import TestCase
```

Comment:
this is fine, but I believe you should also have something like this:

- https://github.com/OpenUpSA/muni-portal-backend/blob/master/.github/workflows/test.yml#L33
- https://github.com/OpenUpSA/municipal-data/blob/master/.github/workflows/test.yml#L29
the literal command a dev is gonna run based on the documentation

And you can do the same with the shapefile:

- wget the shapefile to the CI filesystem
- run the manage command to load it
- check that it works

Is this idea absurd? is it a bad idea? I get a lot of confidence knowing we'll know when this setup stuff breaks, and that I can always have a running system within minutes after `docker compose down --volumes`

Reply:
Added tests for these suggestions. For the shapefile test, I just use the country level - just enough to test the functionality